cvpy/biomedimage/BiomedImage.py

```diff
@@ -78,45 +78,167 @@ def quantify_sphericity(self, image_table: ImageTable, use_spacing: bool, input_
 
         Examples
         --------
-        >>> ## Import classes
+        >>> # Import classes
         >>> from swat import CAS
         >>> from cvpy.base.ImageTable import ImageTable
         >>> from cvpy.biomedimage.BiomedImage import BiomedImage
         >>> from cvpy.biomedimage.LabelConnectivity import LabelConnectivity
         >>> ## Connect to CAS
         >>> s = swat.CAS("example.com", 5570)
-        >>> ## Construct Biomed object
+        >>> # Construct Biomed object
         >>> biomed = BiomedImage(s)
-        >>> ## Construct tables that are paramters to the quantify_sphericity API
+        >>> # Construct tables that are parameters to the quantify_sphericity API
         >>> image_table = s.CASTable(...)
         >>> input_table = ImageTable(image_table)
         >>> output_table = s.CASTable(...)
-        >>> ## Call the API
+        >>> # Call the API
         >>> BiomedImage.quantify_sphericity(input_table.table,....,output_table)
```

Comment:
Nit: If you go back in this file, could you replace "...." (4 dots) with "..." (3 dots).